src/Compatibility/Core/src/Android/Renderers/FontImageSourceHandler.cs

```diff
@@ -5,13 +5,18 @@
 using Android.Graphics;
 using Android.Util;
 using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Controls.Platform;
 using Color = Microsoft.Maui.Graphics.Color;
 using Paint = Android.Graphics.Paint;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace Microsoft.Maui.Controls.Compatibility.Platform.Android
 {
 	public sealed class FontImageSourceHandler : IImageSourceHandler
 	{
+		IFontManager _fontManager;
+		IFontManager FontManager => _fontManager ??= Application.Current?.FindMauiContext()?.Services?.GetService<IFontManager>();
```

Comment:
You could use `imagesource` as the starting point to search for the MauiContext 

`imagesource.FindMauiContext`